src/main/java/poolchange/sdk/sample/main.java

```diff
@@ -89,19 +85,19 @@ public static void main( String[] args )
         Utils.writeConsoleMessage("Creating Azure NetApp Files Account...");
 
         String[] accountParams = {resourceGroupName, anfAccountName};
-        NetAppAccountInner anfAccount = await(CommonSdk.getResourceAsync(anfClient, accountParams, NetAppAccountInner.class));
+        NetAppAccountInner anfAccount = (NetAppAccountInner) CommonSdk.getResource(manager.serviceClient(), accountParams, NetAppAccountInner.class);
         if (anfAccount == null)
         {
             NetAppAccountInner newAccount = new NetAppAccountInner();
             newAccount.withLocation(location);
 
             try
             {
-                anfAccount = await(Creation.createANFAccount(anfClient, resourceGroupName, anfAccountName, newAccount));
+                anfAccount = Creation.createANFAccount(manager.serviceClient(), resourceGroupName, anfAccountName, newAccount);
             }
-            catch (CloudException e)
+            catch (Exception e)
```

Comment:
>Exception [](http://example.com/codeflow?start=19&length=9)

CloudException got replaced by AzureError (at least in Python), and should not be replaced by the broader Exception, can you please apply the equivalent for Java? Please do the same on all samples.